docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-2.md

```diff
@@ -82,7 +82,7 @@ data:
     <html>
       <body>
         This is a common web site.
-        Running in %(loc-name).
+        Running in %(env).
```

Comment:
Why this change?

Reply:
it was a test that accomplished nothing, I still don't know what's wrong with the customization/variable relplaccement

Reply:
You will want to remove this change, it is not helping.